source/module_hamilt_general/module_vdw/vdwd2_parameters.h

```diff
@@ -47,9 +47,9 @@ class Vdwd2Parameters : public VdwParameters
     inline double scaling() const { return scaling_; }
 
   private:
-    double scaling_;
-    double damping_;
-    double radius_;
+    double scaling_=0;
```

Comment:
should be 0.0?

Reply:
This part has been changed in this file.